core/src/main/resources/templates/browseAcls.html

```diff
@@ -584,7 +584,7 @@ <h4 class="card-title" style=" white-space: nowrap;">Topic : {{ topicSelectedPar
 										</td>
 										<td width="5%"></td>
 										<td width="20%" align="right">
-												<a href="requestTopics?topicname={{topicSelectedParam}}&env={{topicPromotionDetails.targetEnvId}}&envName={{topicPromotionDetails.targetEnv}}&requestType=promote">
+												<a href="requestTopics?topicname={{topicSelectedParam}}&env={{topicPromotionDetails.targetEnvId}}&envName={{topicPromotionDetails.targetEnv}}&sourceEnvName={{topicPromotionDetails.sourceEnv}}&requestType=promote">
```

Comment:
```suggestion
												<a href="requestTopics?topicname={{topicSelectedParam}}&env={{topicPromotionDetails.targetEnvId}}&envName={{topicPromotionDetails.targetEnv}}&sourceEnv={{topicPromotionDetails.sourceEnv}}&requestType=promote">
```

as that is id

Reply:
changed with options below to change all of them.